amida-tech/amida-survey-microservice
# Bug/SER-181 Missing export fields

#### What's this PR do?
Adds weight and question index to the fields exported by the survey service

#### Related JIRA tickets:
[SER-181](https://jira.amida-tech.com/browse/SER-181)

#### How should this be manually tested?
1. Generate a survey with a question with a weight answer and responses to that question
1. Request an export from the survey service and check that the response object includes the correct weight values and question indices.

#### Any background context you want to provide?
The tests for `questionIndex` in the export are reduced to testing that the value is a number. It does not test that the value is the correct number because the history object do not include data from the SurveyQuestion table that contains the `line` column.

#### Screenshots (if appropriate):
